### PR TITLE
ipsec manual first release after UI development

### PR DESCRIPTION
**Pull request by cotosso**

### CODE REVIEW

File: ipsec_tunnels.rst [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
+IPsec (IP Security) protocol is the ‘de facto’ standard in VPN tunnels, it’s typically used to create net to net tunnels and it’s supported from all manufacturers. 
+You can use this protocol to create VPN tunnels between a NethSecurity and a device from another manufacturer as well as VPN tunnels between 2 NethSecurity.
 
+NethSecurity by default uses Route-Based VPNs, so each tunnel rely on a specific tun type device. 
+
+Configuration
+-------------
+The configuration of an IPsec tunnel includes 2 peers which we will call A and B which can be:
+
+* 1 Nethsecurity + 1 third-party firewall
```
Comment (gsanchietti) [inline ipsec_tunnels.rst:19]:
```suggestion
* 1 Nethsecurity and 1 third-party firewall
```

File: ipsec_tunnels.rst [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
+* the 2 (or more) networks we want to connect (local network, remote network)
+* the local and remote identifiers (typically the public IPs of the WANs of the 2 firewalls, but others can also be used)
+
+Therefore:
+
+* The WAN IP address of firewall A must coincide with the Remote IP address of firewall B
+* the local network of firewall A must coincide with the remote network of firewall B
+* the local ID of firewall A must coincide with the remote ID of firewall B
+
+All other parameters, however, must be identical in both firewalls to allow correct communication (encryption key, IKE and ESP configuration...).
```
Comment (gsanchietti) [inline ipsec_tunnels.rst:36]:
```suggestion
All other parameters, however, must be identical in both firewalls to allow correct communication (encryption key, IKE and ESP configuration, etc.).
```

File: ipsec_tunnels.rst [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+NethSecurity uses a shared key as the only method to encrypt data.
+
+How to create a new IPsec tunnel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Click :guilabel:`Add IPsec tunnel` button to configure a new tunnel.
+Assign a name to this tunnel then configure it, the drawer is splitted in 3 steps, the step 1/3 contains only network-related parameters, while the others contain all the remaining parameters that must be identical in both firewalls to allow correct communication.
```
Comment (gsanchietti) [inline ipsec_tunnels.rst:42]:
```suggestion
Assign a name to this tunnel then configure it, the drawer is split in 3 steps. The first step contains only network-related parameters, while the others contain all the remaining parameters that must be identical in both firewalls to allow correct communication.
```

File: ipsec_tunnels.rst (hunk 1 above)
Comment (filippocarletti) [reply ipsec_tunnels.rst:19]: Prefer "one" to 1, two to 2, etc if less than 10.

File: ipsec_tunnels.rst (hunk 3 above)
Comment (filippocarletti) [reply ipsec_tunnels.rst:42]: I think that "drawer" is too technical. Maybe "configuration"?

File: ipsec_tunnels.rst [hunk 4]
```diff
@@ -1,14 +1,50 @@
...
+* the local network of firewall A must coincide with the remote network of firewall B
+* the local ID of firewall A must coincide with the remote ID of firewall B
+
+All other parameters, however, must be identical in both firewalls to allow correct communication (encryption key, IKE and ESP configuration, etc.).
+NethSecurity uses a shared key as the only method to encrypt data.
+
+How to create a new IPsec tunnel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Click :guilabel:`Add IPsec tunnel` button to configure a new tunnel.
+Assign a name to this tunnel then configure it, the drawer is splitted in 3 steps, the step 1/3 contains only network-related parameters, while the others contain all the remaining parameters that must be identical in both firewalls to allow correct communication.
```
Comment (cotosso) [inline ipsec_tunnels.rst:42]:
```suggestion
Assign a name to this tunnel then configure it, the configuratinon is split in 3 steps. The first step contains only network-related parameters, while the others contain all the remaining parameters that must be identical in both firewalls to allow correct communication.
```

File: ipsec_tunnels.rst [hunk 5]
```diff
@@ -1,14 +1,50 @@
...
+* the local network of firewall A must coincide with the remote network of firewall B
+* the local ID of firewall A must coincide with the remote ID of firewall B
+
+All other parameters, however, must be identical in both firewalls to allow correct communication (encryption key, IKE and ESP configuration, etc.).
+NethSecurity uses a shared key as the only method to encrypt data.
+
+How to create a new IPsec tunnel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Click :guilabel:`Add IPsec tunnel` button to configure a new tunnel.
+Assign a name to this tunnel then configure it, the drawer is split in 3 steps. The first step contains only network-related parameters, while the others contain all the remaining parameters that must be identical in both firewalls to allow correct communication.
```
Comment (cotosso) [inline ipsec_tunnels.rst:42]:
```suggestion
Assign a name to this tunnel then configure it, the configuration is split in 3 steps. The first step contains only network-related parameters, while the others contain all the remaining parameters that must be identical in both firewalls to allow correct communication.
```

Comment (cotosso) [reply ipsec_tunnels.rst:42]: @filippocarletti

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**